projects/bp-gallery/src/hooks/generic-endpoints.hook.ts

```diff
@@ -30,34 +38,72 @@ const useGenericTagEndpoints = (type: TagType) => {
         return {
           allTagsQuery: useGetAllLocationTagsQuery,
           tagsWithThumbnailQuery: useGetLocationTagsWithThumbnailQuery,
+          tagPictures: useGetPicturesForLocationQuery,
           updateTagNameMutationSource: useUpdateLocationNameMutation,
           updateSynonymsMutationSource: useUpdateLocationSynonymsMutation,
+          updateVisibilityMutationSource: useUpdateLocationVisibilityMutation,
+          updateTagParentMutationSource: useUpdateLocationParentMutation,
+          updateTagAcceptanceMutationSource: useUpdateLocationAcceptanceMutation,
+          updateTagChildMutationSource: useUpdateLocationChildMutation,
+          updateRootMutationSource: useUpdateLocationRootMutation,
           mergeTagsMutationSource: useMergeLocationTagsMutation,
           deleteTagMutationSource: useDeleteLocationTagMutation,
-          updateVisibilityMutationSource: useUpdateLocationVisibilityMutation,
+          createTagMutationSource: useCreateLocationTagMutation,
         };
       case TagType.PERSON:
         return {
           allTagsQuery: useGetAllPersonTagsQuery,
           tagsWithThumbnailQuery: useGetPersonTagsWithThumbnailQuery,
+          tagPictures: () => {
+            return { data: undefined };
+          },
           updateTagNameMutationSource: useUpdatePersonNameMutation,
           updateSynonymsMutationSource: useUpdatePersonSynonymsMutation,
-          mergeTagsMutationSource: useMergePersonTagsMutation,
-          deleteTagMutationSource: useDeletePersonTagMutation,
           updateVisibilityMutationSource: (dummy: any) => {
             return [dummy];
           },
+          updateTagParentMutationSource: (dummy: any) => {
+            return [dummy];
+          },
+          updateTagAcceptanceMutationSource: (dummy: any) => {
+            return [dummy];
+          },
+          updateTagChildMutationSource: (dummy: any) => {
+            return [dummy];
+          },
+          updateRootMutationSource: (dummy: any) => {
```

Comment:
maybe don't return `[dummy]` but rather call the parameter `_dummy` to avoid the linter error? same below